proposals/pr-contrib-guide.md

```diff
@@ -0,0 +1,496 @@
+
+Contributions made without a GitHub issue will not be prioritized to move to the
+next stage. Pull requests that are not linked to an issue and are more than six
+months old will be considered stale and closed. See the
+[rationale](#rationale-require-issues) for further discussion.
+
+### Ready for review stage
+
+Once the preliminary review criteria from the implementation stage have been
+met, the tracking GitHub issue is labeled as ready for review and added to a
+queue for maintainer review. The maintainers will select issues for full review
+from this queue. The issues will be prioritized for review publicly based on the
+following criteria in descending order of importance:
+
+1. Bugs, documentation changes, and feature requests will be prioritized in that
+   order.
+1. Older contributions will be selected before more recent ones.
+
+A maintainer must select the issue from the queue with the current highest
+priority. The prioritization of issues in the queue is reviewed on a regular
+cadence and is ultimately decided based on the maintainers' discretion.
+
+### Review stage
+
+Selecting a pull request for review will move the tracking GitHub issue to the
+under review stage. In this stage, the maintainers will work with the
+contributor to perform a full review of the contribution with the goal of
+merging the pull request. Having a pull request selected for review does not
+guarantee how long it will be until it is merged. The maintainers will review no
+more than four pull requests at a time. The criteria for merging a pull request
+may include:
+
+1. The change is made in the right place and fits in the existing architecture.
+1. The change does not increase the maintainability in an unreasonable fashion.
+1. There is sufficient documentation, both in the code and for end users.
+1. The code is readable and follows best practices and conventions.
+
+If a pull request is not ready to merge, the maintainers will comment on the
+pull request with their feedback and questions for the contributor to address.
+If changes are requested, the contributor should make the changes within 15
+business days. If the contributor does not respond within 15 days, the pull
+request will be marked as stale and the contributor will have 5 days to respond.
+If there is no response, the maintainers will complete the pull request.
+
+Once a pull request is completed to the satisfaction of the maintainers, it will
+be approved and merged. The corresponding tracking issue will be closed closed
+and labeled to acknowledge it as a community contribution.
+
+## Contribution Kanban board
+
+The contribution process will be implemented with a publicly visible [GitHub
+project Kanban
+board](https://docs.github.com/en/issues/organizing-your-work-with-project-boards).
+This project board will track GitHub issues from the contribution-ready stage
+through completion by merging a pull request. Each lane of the project board
```

Comment:
It would be great to add a link to the sample Kanban board you have early on to have a visualization to follow, especially since I'm not sure how many people are familiar with GitHub projects.